content/en/docs/installation/uninstall.md

```diff
@@ -25,3 +25,45 @@ Delete the `NOTATION_LIBEXEC` directory to remove the additional binaries, inclu
 **IMPORTANT:** `notation` automatically creates the `NOTATION_CONFIG` directory if it does not exist. If you remove the `NOTATION_CONFIG` directory, `notation` will recreate the directory the next time you run a command.
```

Comment:
Can we put a link to show the users what is `NOTATION_CONFIG` directory?

Reply:
Line 13 already addresses this: https://github.com/notaryproject/notaryproject.dev/pull/164/files#diff-ebf28445e02ae412163e2fde43e76bee51e880abab8449ac8b4b863fec8d4354L13